modules/reference/pages/cluster-properties.adoc

```diff
@@ -293,6 +293,46 @@ Max number of simultaneous connections to S3 per shard. Includes connections use
 
 ---
 
+=== cloud_storage_recovery_topic_validation_mode
+
+Validation mode performed before recovering a topic from cloud storage. This property accept the following parameters:
+
+- `no_check`. Skips the checks for topic recovery.
+- `check_manifest_existence`.  Run an existence check of each `partition_manifest`. Fails if there are connection issues to the cloud storage.
+- `check_manifest_and_segment_metadata`. Download the manifest and run a consistency check, comparing the metadata with the cloud storage objects. The process will fail if any metadata references cloud storage objects that are missing.
+
+*Requires restart:* No
+
+*Required:* Yes
```

Comment:
@andijcr is this property required? From the autogen script, we got null: no.

Reply:
No it's not. there is already a default value